.github/workflows/argos.yml

```diff
@@ -0,0 +1,36 @@
+name: Argos CI Screenshots
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  take-screenshots:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v4
+
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: current

```

Comment:
The `current` v23 will be gone in April and can introduce bugs that bjork the pipeline.

personally I'd go with `v22`/`22`, but if you find a `lts` option for this, that would be fantastic too.